### PR TITLE
added animated sprite. refactored grid code.

### PR DESCRIPTION
**Pull request by gabrilend**
Improved the grid generation from a static set of statements to a dynamic algorithm. We should be able to utilize dynamically sized grids now, should we decide to. If nothing else it will be useful when iterating in the playtesting phase.

### CODE REVIEW

File: sprites/characters/protagonist_idle.zip
Comment (elainajones) [inline sprites/characters/protagonist_idle.zip:1]: I don't think we want to track this.

**Review by elainajones — CHANGES_REQUESTED** [review]
Please remove the zip archive. This might have been added in error :)

File: sprites/characters/protagonist_idle.zip
Comment (gabrilend) [reply sprites/characters/protagonist_idle.zip:1]: Good catch.

**Comment by gabrilend** [comment]
zip file removed.

**Review by elainajones — APPROVED** [review]

**Pull request merged**